mne/gui/help/kit2fiff.json

```diff
@@ -1,7 +1,10 @@
 {
+  "hsp_file": "*.hsp or *.txt file containing digitized head shape points",
+  "fid_file": "*.elp or *.txt file containing 8 digitized fiducial points (\"electrode laser points\")",
```

Comment:
should we be calling this `fid_file`? only the first three points are typically referred to as fiducials? wouldn't `elp_file` be a better name?